keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -312,6 +340,31 @@ func (runtime *CRIRuntime) RunMQTT(mqttImage string) error {
 	return runtime.RuntimeService.StartContainer(containerID)
 }
 
+func (runtime *CRIRuntime) RemoveMQTT() error {
+	sandboxFilter := &runtimeapi.PodSandboxFilter{
```

Comment:
Another question, is it needed to use cri for create a container in edgenode? Or remote runtime, we can only use the cri. cc @zc2638

Reply:
Sorry, I may not have understood what you meant.
What you're trying to say is that whatever "remote runtime" is, just consider using CRI, not docker?

Reply:
I mean there's no need to use cri for creating mqtt container. Whether can we just use  the oci api to create?

Reply:
OCI includes runtime spec, image spec, and distribution spec, but I can't find the corresponding interface in opencontainers. 
Can I see the available associated interfaces?